### DISCUSSION TITLE
Akira, the successes and struggles of building a UX application from scratch

### DISCUSSION THREAD

**Discussion by Alecaddd** [elementary/edw#32] [Accepted Talks]
Speaker: @alecaddd

### Description
A quick overview of the initial struggles and motivations behind the launching of the project, followed by a quick analysis of the stack currently used, the state of the application, and future plans.

**Comment by cassidyjames** [comment]
@Alecaddd I'm not sure if you saw, but we've accepted your talk. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.
